io_ogre/ogre/mesh.py

```diff
@@ -240,7 +240,7 @@ def dot_mesh( ob, path, force_name=None, ignore_shape_animation=False, normals=T
                     _remap_normals_.append( n )
                     _face_indices_.append( F.index )
 
-                    x,y,z = swap(ob.matrix_world * v.co)        # xz-y is correct!
```

Comment:
is this change on purpose?

Reply:
hmmmmmmmmmmm .. I don't remember making this change ...